### ISSUE TITLE
Test set size mismatch: Available dataset (2,446) vs AICrowd evaluation (3,610)

### ISSUE THREAD

**Issue by akramsalim** [talk2car/Talk2Car#19]
## Issue Description
There appears to be a mismatch between the available Talk2Car test set and the expected size for AICrowd submission.

## Details
- **Available test set size:** 2,446 samples (latest version downloaded)
- **AICrowd C4AV challenge expects:** 3,610 samples
- **Error message:** "You have 2439 but you should have 3610 samples "

## Questions
1. Which version of the Talk2Car test set should be used for the ECCV 2020 C4AV challenge?
2. Is there an extended test set with 3,610 samples that we should be using?
3. Has the test set been updated since the original challenge?

## Context
Attempting to submit predictions to: https://www.aicrowd.com/challenges/eccv-2020-commands-4-autonomous-vehicles

Any clarification would be greatly appreciated!

**Comment by ThierryDeruyttere** [comment]
Hey,

Thanks for the interest in Talk2Car.
The test set on AICrowd is actually the test set + the validation set because at the time of the challenge we wanted to see if people were also training on the validation set to get better results.

But since a couple of years we have also released the talk2car test set ground truth i believe so you don’t need to use AICrowd anymore.

Hope this helps.



Op 31 jul 2025 om 01:21 heeft akramsalim ***@***.***> het volgende geschreven:

﻿
[https://avatars.githubusercontent.com/u/60445499?s=20&v=4]akramsalim created an issue (talk2car/Talk2Car#19)<https://github.com/talk2car/Talk2Car/issues/19>
Issue Description

There appears to be a mismatch between the available Talk2Car test set and the expected size for AICrowd submission.

Details

  *   Available test set size: 2,446 samples (latest version downloaded)
  *   AICrowd C4AV challenge expects: 3,610 samples
  *   Error message: "You have 2439 but you should have 3610 samples"

Questions

  1.  Which version of the Talk2Car test set should be used for the ECCV 2020 C4AV challenge?
  2.  Is there an extended test set with 3,610 samples that we should be using?
  3.  Has the test set been updated since the original challenge?

Context

Attempting to submit predictions to: https://www.aicrowd.com/challenges/eccv-2020-commands-4-autonomous-vehicles

Any clarification would be greatly appreciated!

**Comment by ThierryDeruyttere** [comment]
I'm closing this for now.
